components/style/values/specified/box.rs

```diff
@@ -235,6 +273,7 @@ impl Display {
     #[inline]
     fn from3(outside: DisplayOutside, inside: DisplayInside, list_item: bool) -> Self {
         let inside = match inside {
+            #[cfg(any(feature = "servo-layout-2013", feature = "gecko"))]
```

Comment:
Why not leaving it computing `flow` to `inline` / `block` as everywhere else?

Reply:
This is compiled out because I removed a lot of variants from `DisplayInside`, most notably `Block` and `Inline`.